requirements.txt

```diff
@@ -1,7 +1,7 @@
 aiohttp==2.2.5
 async-timeout==2.0.0
 chardet==3.0.4
-discord.py==1.0.0a1393+98a054e
+https://github.com/Rapptz/discord.py/archive/master.zip?1#egg=discord.py[voice]
```

Comment:
we're not using [voice] features right now, so this should just be the standard egg

Reply:
voice stuff planned for later, installing by requirements.txt fails with that version string

also i love docker